mmv1/third_party/terraform/website/docs/r/compute_instance.html.markdown

```diff
@@ -411,6 +411,10 @@ specified, then this instance will have no external IPv6 Internet access. Struct
 
 * `values` (Required) - The values for the node affinity label.
 
+<a name="nested_params"></a>The `params` block supports:
```

Comment:
We'll need to document initialize_params.resource_manager_tags as well as include the top-level `params` field!

Reply:
Added.